### DISCUSSION TITLE
Newbie : Apple Sample Code / Swift Composable Architecture

### DISCUSSION THREAD

**Discussion by nonfungibletunji** [pointfreeco/swift-composable-architecture#512] [Q&A]
Is there any blog post or information on how to convert Apple sample code, Fruta, to the SCA?

**Comment by andreyz** [comment]
I'm afraid there isn't. But there's plenty of well commented sample code in the TCA package itself with SwiftUI and UIKit examples for almost all basic app components and interactions.

[PointFree ComposableArchitecture](https://www.pointfree.co/collections/composable-architecture) collection of articles is an amazing place to start from (some require a subscription to access).
